wercker.yml

```diff
@@ -0,0 +1,10 @@
+box: centos:6
+build:
+  steps:
+    - script:
+        name: install rmagick
+        code: |
+          sudo yum install -y ruby ruby-devel rubygems ImageMagick-devel gcc
```

Comment:
I think ideally we would, but maybe just checking that it installs would be a good start. I think it would be good to confirm that it would fail the build if it breaks on CentOS, though. There was a version of the gem that did break, right? Could we try running the build with `sudo gem install rmagick -v=whatevs`?